engine/src/configuration/configuration.h

```diff
@@ -0,0 +1,95 @@
+#ifndef CONFIGURATION_H
+#define CONFIGURATION_H
+
+
+struct Configuration
+{
+    // Config Structs Declaration
+
+    // These are settings that are relevant to collision.cpp and are marked as hacks
+    // or their comments are unclear or they are unclear
+    struct CollisionHacks {
+        const float collision_hack_distance;
+        const bool collision_damage_to_ai;
+        const bool crash_dock_hangar;
+        const bool crash_dock_unit;
+        const float front_collision_hack_angle;
+        const float front_collision_hack_distance;
+        const bool cargo_deals_collide_damage;
+        //const int upgrade_faction = (cargo_deals_collide_damage ? -1 : FactionUtil::GetUpgradeFaction())
+
+        CollisionHacks();
+    };
+
+    // Also radar
+    struct Computer {
+        // Note: below is probably a stale comment
+        // DO NOT CHANGE see unit_customize.cpp
+        const float default_lock_cone;
+        const float default_max_range;
+        const float default_tracking_cone;
+
+        Computer();
+    };
+
+    struct Fuel {
+        const float afterburner_fuel_usage;
+        const float fmec_exit_velocity_inverse; // Well, that's very clear
+        const float fuel_efficiency;
```

Comment:
There's some relevant context that's been removed from the original name of this variable "Lithium6constant" -- a better name would ultimately be "FuelToEnergyConversionRelativeToLithium6DeuterideFusion" - that fully encodes what the efficiency is relative to. This isn't a request to change anything, just writing this down while it's fresh(ish) in my head. I plan on doing a pass at some point to go through all the config vars adding comments, invariants, SI units, etc. and changing names if appropriate.

Reply:
To clarify - Lithium6constant isn't a good name for what it is - but it's one that's easy for me to identify and subsequently fix. I'd prefer if we limited the renaming of thing prior to consolidation